.github/workflows/ci.yaml

```diff
@@ -508,4 +508,38 @@ jobs:
         run: |
           . venv/bin/activate
           pip install -e .
-          pytest -m primer_stdlib -n auto
+          pytest -m primer -n auto -k test_primer_stdlib_no_crash
+
+  pytest-primer-external:
```

Comment:
IMO it would make sense to create a new workflow file for it. That will make it easier to distinguish for all the other tests in the Github UI.

Only small downside, you need to compy the `prepare-tests-linux` job.

Reply:
#5359